### ISSUE TITLE
Router's host name vs. model in the email subject line

### ISSUE THREAD

**Issue by dorner365** [maverickcdn/wicens#8]
The script works well and is useful. Thank you for that!

A 'nice to have' (from my perspective) would be showing the host name in the email subject line instead of the router's model. Or the option to chose one or the other.

Currently the router's IP in the message body is the only difference for same model routers.

**Comment by maverickcdn** [comment]
To clarify, are you saying the host name (device name) in the subject line in all Email notification types? or just a specific one? (WAN IP change/Firmware update/Reboot/Script update)

Having a selection of either is alot of recoding, simply adding the hostname should be easily accomplished

**Comment by dorner365** [comment]
Yes, the host name (device name) in the subject line of all notification types. 
    On Monday, August 19, 2024 at 05:35:51 p.m. EDT, maverickcdn ***@***.***> wrote:  
 
 


To clarify, are you saying the host name (device name) in the subject line in all Email notification types? or just a specific one? (WAN IP change/Firmware update/Reboot/Script update)

Having a selection of either is alot of recoding, simply adding the hostname should be easily accomplished

**Comment by maverickcdn** [comment]
Sorry for the delay, this change has been made
